xtra-libp2p-offer/src/maker.rs

```diff
@@ -59,9 +57,12 @@ impl Actor {
     async fn handle(&mut self, msg: NewOffers, ctx: &mut xtra::Context<Self>) {
         self.latest_offers = msg.0;
 
+        let quiet = tracing::info_span!(target: "quiet_children", "sometimes_quiet_children");
```

Comment:
consider creating a factory function in `shared_bin` that just returns this, maybe named `quiet_span()` ?

then you can use it without leaking it implementation details